### ISSUE TITLE
prettier-tslint mangles imports when reordering

### ISSUE THREAD

**Issue by kbd** [azz/prettier-tslint#19]
Environment:
* VS Code 1.27.2
* Prettier VS Code extension
* prettier-tslint
* `"module": "commonjs"` in tsconfig.json
* `"ordered-imports": true` in tslint.json

Steps taken:

Created `testing.ts` with the following in it:

```typescript
import crypto from 'crypto'
import path from 'path'
import fs from 'fs'
```

Upon calling "Format Document" in VS Code, it gets mangled to:

```typescript
import crypto from 'crypto'
import fs from "f'path'ort path from "p'fs'
```

Correct is:

```typescript
import crypto from 'crypto'
import fs from 'fs'
import path from 'path'
```

The vscode-tslint extension with `tslint.autoFixOnSave` set to `true` doesn't have this bug and correctly reorders imports on save.

**Comment by aleclarson** [comment]
This is probably a `tslint` issue, not `prettier-tslint`.

**Comment by aleclarson** [comment]
Let me know if you can reproduce this issue still. Fork [this repository](https://github.com/aleclarson/repro/tree/prettier-tslint-19) to start with.

**Comment by aleclarson** [comment]
Looks like many errors were being caused by an undefined `parent` property on AST nodes, which has now been fixed by e06519376f9acb30d6e05e02be49a4505d0b7dd2 (which will be released soon).
